drools/pom.xml

```
@@ -35,11 +35,30 @@
     <module>kogito-pmml-openapi</module>
     <module>kogito-pmml</module>
     <module>kogito-dmn</module>
-    <module>drools-decisiontables</module>
     <module>kogito-drools</module>
     <module>kogito-scenario-simulation</module>
   </modules>
 
+  <profiles>
+    <profile>
+      <id>community</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <modules>
+        <module>drools-decisiontables</module>
+      </modules>
+    </profile>
+    <profile>
+      <id>prod</id>
+      <activation>
+        <property>
+          <name>prod</name>
```

i would use the same property we use in other assembllies
```suggestion
          <name>productized</name>
```